tests/foreman/sys/test_rename.py

```diff
@@ -43,6 +44,8 @@ def test_positive_rename_satellite(self, module_org, module_product):
         :setup: Satellite server with synchronized rh and custom
             repos and with a registered host
 
+        :bz: 1901983
+
```

Comment:
@pondrejk the issue seems similar to https://bugzilla.redhat.com/show_bug.cgi?id=1897360